adam-cli/src/main/scala/org/bdgenomics/adam/cli/Adam2Fastq.scala

```diff
@@ -70,7 +70,7 @@ class Adam2Fastq(val args: Adam2FastqArgs) extends BDGSparkCommand[Adam2FastqArg
       else
         None
 
-    var reads: RDD[AlignmentRecord] = sc.loadAlignments(args.inputPath, projection = projectionOpt)
+    var reads: RDD[AlignmentRecord] = sc.loadAlignments(args.inputPath, projection = projectionOpt)._1
```

Comment:
I would rather this primary API entry point didn't return a tuple.  How about a new load method that returns the tuple, and keep this one returning the RDD?